cardinal/component_test.go

```diff
@@ -32,11 +43,24 @@ func TestComponentExample(t *testing.T) {
 	assert.NilError(t, cardinal.RegisterComponent[Height](world))
 	assert.NilError(t, cardinal.RegisterComponent[Weight](world))
 	assert.NilError(t, cardinal.RegisterComponent[Age](world))
+	assert.NilError(t, cardinal.RegisterComponent[Number](world))
 
 	testWorldCtx := testutils.WorldToWorldContext(world)
 	startHeight := 72
 	startWeight := 200
 	startAge := 30
+	numberId, err := cardinal.Create(testWorldCtx, &Number{})
```

Comment:
Check this test here @jerargus

Reply:
https://linear.app/arguslabs/issue/WORLD-488/getcomponent-still-working-after-removal-of-entity